packages/st2/rpm/st2.spec

```diff
@@ -13,9 +13,24 @@ Epoch: %{epoch}
 
 %if 0%{?use_st2python}
 Requires: st2python, python-devel, openssl-devel, libffi-devel, git, pam, openssh-server, openssh-clients, bash, setup
-%else
+%if 0%{?rhel} == 7
 Requires: python-devel, openssl-devel, libffi-devel, git, pam, openssh-server, openssh-clients, bash, setup
+%if 0%{?rhel} >= 8
+Requires: python3-devel openssl-devel, libffi-devel, git, pam, openssh-server, openssh-clients, bash, setup
+%endif
 %endif
+%endif
+
+# EL8 requires a few python packages available within 'BUILDROOT' when outside venv
+# These are in the el8 packagingbuild dockerfile
+# Reference https://fossies.org/linux/ansible/packaging/rpm/ansible.spec
+%if 0%{?rhel} >= 8
+# Will use the python3 stdlib venv
+BuildRequires: python3-devel
+BuildRequires: python3-setuptools
+Requires: python3-virtualenv
```

Comment:
Any additional context why this requirement is needed as install dependency for `st2` rpm package?
I guess virtualenv is what we already bundle within st2 package.
What happens if we remove this dependency?

Reply:
So - you need the virtualenv executable present. I will test it out to see if it isn't needed.

Reply:
I removed line 31. L29-30 are needed for the rpmbuild environment.